presentation/src/main/java/com/globant/equattrocchio/cleanarchitecture/mvp/view/ImagesView.java

```diff
@@ -14,14 +20,19 @@
 public class ImagesView extends ActivityView {
 
     @BindView(R.id.tv_incoming_json) TextView tvlabel;
+    @BindView(R.id.recycler_images) RecyclerView recyclerImages;
 
     public ImagesView(AppCompatActivity activity) {
         super(activity);
         ButterKnife.bind(this, activity);
     }
 
-    public void showText(String text) {
-        tvlabel.setText(text);
+    public void loadRecycler(Result result) {
+        tvlabel.setVisibility(View.GONE);
+        LinearLayoutManager mLayoutManager = new LinearLayoutManager(getContext());
```

Comment:
You can save `getContext()` in a variables as you are using it more that once. Also you need to perform a non null validation for `context`. If `context` is null, you just simply add `return;`.